.github/workflows/manual_tests.yml

```diff
@@ -0,0 +1,92 @@
+name: Tests on network
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: 'Network to run tests'
+        default: 'integration'
+        required: true
```

Comment:
Would it make sense to define allow running this only on `integration` and `testnet` using this? 
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/